config/routes.rb

```diff
@@ -1,14 +1,17 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  get 'groups/index'
+  get 'groups/show'
```

Comment:
Why are you using non-resourceful routes?
please remove the autogenerated routes if they do not have any use.

Reply:
Noted. Done. Thanks.